dandi/organize.py

```diff
@@ -348,6 +359,8 @@ def _get_unique_values_among_non_unique(metadata, non_unique_paths, field):
 
 def get_obj_id(object_id):
     """Given full object_id, get its shortened version"""
+    import numpy as np
```

Comment:
`potential_fields` now needs to be imported from `organize.py` in order to be used in the `--required-field` definition, but that leads to a bunch of unconditional "heavy" imports unless this line (and others) is moved from the module-level to function-level.

Reply:
Commit just says `Fix heavy imports` but what is "fixed" since it was there before -- is there a test fail or what?

Indeed for `dandi * --help` we might want to avoid heavy imports (but then should be done at the imports of `cmd_` files) -- I would not bother trying to delay such a generic import as `numpy`.  Not yet sure about other imports

Reply:
Yes, [this test](https://github.com/dandi/dandi-cli/blob/8e67482002257d47017e1cbb6234298970ea383c/dandi/cli/tests/test_command.py#L38) was failing.

Reply:
I see - thanks!  So let's move `potential_fields` into `consts.py` as `dandi_layout_fields` variable, import from there where needed. `.consts` is a lightweight import.  And revert this imports fixing commit.

Reply:
Done.